src/app/images/components/ImagesForms/SelectUpstreamImagesForm/SelectUpstreamImagesForm.tsx

```diff
@@ -199,7 +199,10 @@ const SelectUpstreamImagesForm = (): ReactElement => {
   };
 
   return (
-    <>
+    <div
+      className="select-upstream-images-form"
+      data-testid="select-upstream-images-form"
```

Comment:
Unless this testid is strictly needed for a test and there's no other way of testing using accessible attributes, I would remove this.